packages/components-angular/src/app/examples/checkbox-example.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'page-checkbox-example',
+  template: `
+    <form (submit)="onSubmit($event)">
+      <p-checkbox name="some-name" label="Some Label" value="some-value"></p-checkbox>
```

Comment:
```suggestion
      <p-checkbox name="some-name" label="Some Label" value="some-value" />
```

Could be used self closing?